sebs/aws/aws.py

```diff
@@ -198,12 +345,17 @@ def create_function(self, code_package: Benchmark, func_name: str) -> "LambdaFun
                 self.config.resources.lambda_role(self.session),
                 function_cfg,
             )
+            if container_deployment:
+                code_package = container_uri
             self.update_function(lambda_function, code_package)
             lambda_function.updated_code = True
+            # PK: TO DO: Not sure if update function needs to be handled when performing container_deployment 
```

Comment:
That's an excellent question - please check AWS docs. I think that we need to do an update of configuration. When deploying new container to AWS Fargate, we had to create a new version of the application to ensure that it uses the newest container image.

Reply:
I checked as of now lambda does pull the latest image. I reckon it is because we are doing the update_function_config whenever we do the update_function either through the code/s3/ECR

Reply:
Sounds good - if we update the config, we should force Lambda to redeploy which would trigger using new image everywhere.